azure-pipelines.yml

```diff
@@ -0,0 +1,27 @@
+jobs:
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-20.04'
+  steps:
+  - script: |
+      conda update -y -q conda
+      conda create -q -y -n  -c conda-forge bench python=3.7 pandas daal4py scikit-learn xgboost

```

Comment:
Here we use "-c conda-forge". But in other places we recommend "-c intel" to download daal4py. 
Do we want to use conda-forge everywhere?

Reply:
I want to use PyPi everywhere :) Yes, I think _conda-forg_e are preferable in perspective

Reply:
But pip in unavailable right now, so let's use conda-forge until releasing pip.